sklearn/multioutput.py

```diff
@@ -365,3 +368,232 @@ def score(self, X, y):
                              format(n_outputs_, y.shape[1]))
         y_pred = self.predict(X)
         return np.mean(np.all(y == y_pred, axis=1))
+
+
+class ClassifierChain(BaseEstimator):
+    """A multi-label model that arranges binary classifiers into a chain.
+
+    Each model makes a prediction in the order specified by the chain using
+    all of the available features provided to the model plus the predictions
+    of models that are earlier in the chain.
+
+    Parameters
+    ----------
+    base_estimator : estimator
```

Comment:
What are your thoughts on allowing arbitrary lists of classifiers? For instance, one could use a logistic regression for one task, a Random Forest classifier for another, etc.. This is not something that would need to be implemented for this PR to get merged, I'm just asking out of curiosity.